src/app/components/form/DualCurrencyField/index.tsx

```diff
@@ -125,7 +138,12 @@ export default function DualCurrencyField({
         )}
       </div>
       {hint && (
-        <p className="my-1 text-xs text-gray-700 dark:text-neutral-400">
+        <p
+          className={classNames(
+            "my-1 text-xs text-gray-700 dark:text-neutral-400",
+            !!amountExceeded && "text-red-500 dark:text-red-500"
```

Comment:
why is the dark: version the same? can't it be omitted? (multiple places in this file)

Reply:
no, because the dark:text-neutral-400 class supersedes the dark:text-red-500 if it's not set.